integration/app_integration_test.go

```diff
@@ -871,7 +957,7 @@ func setupWithOptions(t *testing.T, opts appTestOptions) *pack {
 	rcConf := service.MakeDefaultConfig()
 	rcConf.Console = nil
 	rcConf.Log = log
-	rcConf.DataDir, err = ioutil.TempDir("", "cluster-"+p.rootCluster.Secrets.SiteName)
+	rcConf.DataDir, err = os.MkdirTemp("", "cluster-"+p.rootCluster.Secrets.SiteName)
```

Comment:
As far as I understand, each cluster needs its own `DataDir,` so we create a temporary directory based on `SiteName` [(which is the `ClusterName`)](https://github.com/gravitational/teleport/blob/master/integration/helpers.go#L251). I can update this line actually to use the `ClusterName`. Do you think it will be clear?

Reply:
We can use it since every call returns a different directory.

From [TempDir documentation](https://pkg.go.dev/testing#T.TempDir):
> Each subsequent call to t.TempDir returns a unique directory;